src/components/shared/discreetMode/discreetMode.js

```diff
@@ -38,13 +44,15 @@ class DiscreetMode extends Component {
 
 DiscreetMode.defaultProps = {
   addresses: [],
+  shouldEvaluate: false,
```

Comment:
💅 `shouldEvaluate` is not telling me what is the flag doing.

I find it really hard to find a better name. Best I got so far is:
```suggestion
  blurShouldBeDisabledOnOtherAccounts: false,
```

If you can think of some even clearer name, I'd be happy to accept it.

Reply:
Thx Vit for the review, you are right the prop name is for evaluate or check that logic, so I just changed.

About your concern for the behavior of other wallet account, I checked with Ali again and yes it is fine that when you see other account you can don't see any amount blur, so the behavior is still the same.